playbooks/symfony/symfony-messenger.md

```diff
@@ -0,0 +1,34 @@
+---
+title: When to use a Symfony Messenger component 🚚
+description: A brief guideline on when to use a Symfony Messenger component
+---
+
+Symfony Messenger is a very powerful component that allows you to implement some level of concurrency in your
```

Comment:
As mentioned in https://github.com/bornfight/README/pull/45/files#r422105498, i believe that concurrency is not main idea behind messenger, but rather separation of concerns resulted by implementing something like message bus / command bus. Concurrency is simply a way in which you can process messages given you need it. But i guess that really depends on project, so this is not wrong in any sense.

Cheers 🍻